fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java

```diff
@@ -86,6 +96,44 @@ private void writeUnlock() {
         lock.writeLock().unlock();
     }
 
+    @Override
+    protected void runAfterCatalogReady() {
+        List<ExportJob> jobs = getExportJobs(JobState.PENDING);
+        // Because exportJob may be replayed from log
+        // we also need handle EXPORTING state exportJob.
+        jobs.addAll(getExportJobs(JobState.EXPORTING));
```

Comment:
I think we don't need to handle `exporting` state jobs here.
All jobs replayed from log should be reset to PENDING.